### ISSUE TITLE
Paru does not offer to install missing dependencies on AUR package upgrade

### ISSUE THREAD

**Issue by Porkepix** [Morganamilo/paru#1182]
### Affected Version

paru v2.0.3 - libalpm v14.0.0

### Description
**Have you checked previous issues?** Yes

On upgrading an AUR package with missing dependencies (here specifically build dependencies), `paru` won't offer to install (and remove them) to build the package and will straight fail with the following message:

`error: can't build foo-0.20.0-2, deps not satisfied: bar`

### Output

No full output as it really wouldn't be relevant here. Configuration files are very close to default, and nothing should be related to the build dependencies. I can of course quickly add anything additional you might find relevant.

```
fetching devel info...
error: can't build foo-0.20.0-2, deps not satisfied: bar
error: packages failed to build: foo-0.20.0-2
```

**Comment by nabaco** [comment]
I have the same issue. Installing the dependencies manually and then installing the required packages works
```
paru --version
paru v2.0.4 - libalpm v15.0.0
```

**Comment by andruhon** [comment]
I have similar issue when installing updates and getting stuck on missing `patchelf` when updating `bambustudio-bin`

**Comment by Beethoven-n** [comment]
having this issue with `drop-oss-app` too. tried to add a new dependency that wasn't listed in the PKGBUILD before but it won't go download it

**Comment by Beethoven-n** [comment]
yeah i'm gonna just do `makepkg -src` in file manager review mode and move on with my life
